modules/op-release-notes-1-6.adoc

```diff
@@ -0,0 +1,194 @@
+// Module included in the following assembly:
+//
+// * cicd/pipelines/op-release-notes.adoc
+
+[id="op-release-notes-1-6_{context}"]
+= Release notes for {pipelines-title} General Availability 1.6
+
+With this update, {pipelines-title} General Availability (GA) 1.6 is available on {product-title} 4.7, 4.8 and 4.9.
+
+[id="compatibility-support-matrix-1-6_{context}"]
+== Compatibility and support matrix
+
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use.
+
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
+
+In the table, features are marked with the following statuses:
+
+- *TP*: _Technology Preview_
+
+- *GA*: _General Availability_
+
+Note the following scope of support on the Red Hat Customer Portal for these features:
+
+.Compatibility and support matrix
+[cols="1,1,1",options="header"]
+|===
+| Feature   | Version       | Support Status
+| {product-title} | 4.9     | GA
+| Pipelines Operator | 1.6  | GA
+| Pipelines | 0.28.x        | GA
+| CLI       | 0.21.x        | GA
+| Catalog   | 0.28          | GA
+| Triggers  | 0.16.x        | GA
+| Pipeline resources | -    | TP
+|===
+
+For questions and feedback, you can send an email to the product team at pipelines-interest@redhat.com.
+
+[id="new-features-1-6_{context}"]
+== New features
+
+In addition to the fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.6.
+
+=== CLI 0.21.0 release
+
+* With this update, you can configure a pipeline or task `start` command to return a YAML or JSON-formatted string by using the `--output <string>`, where `<string>` is `yaml` or `json`. Otherwise, without the `--output` option, the `start` command returns a human-friendly message that is hard for other programs to parse. Returning a YAML or JSON-formatted string is useful for continuous integration (CI) environments.  For example, after a resource is created, you can use `yq` or `jq` to parse the YAML or JSON-formatted message about the resource and wait until that resource is terminated without using the `showlog` option. (link:https://github.com/tektoncd/cli/pull/1326[#1326])
+
+* With this update, you can authenticate to a registry with Podman's `auth.json` authentication file. For example, you can use `tkn bundle push` to push to a remote registry using podman instead of docker CLI. (link:https://github.com/tektoncd/cli/pull/1430[#1430])
+
+* With this update, if you use the `tkn taskrun delete --all` command, you can preserve runs that are younger than a specified number of minutes by using the new `--keep-since <minutes>` option. For example, to keep task runs that are less than five minutes old, you enter `$ tkn taskrun delete -all --keep-since 5`. (link:https://github.com/tektoncd/cli/pull/1435[#1435])
```

Comment:
Consider maybe laying this out so that it's easier to copy the commands?
e.g.
```
* With this update, you can use the `--keep-since <minutes>` flag with the `tkn taskrun delete --all` command to preserve runs that are younger than a specified number of minutes (link:https://github.com/tektoncd/cli/pull/1435[#1435]).
+
.Example command to keep task runs that are less than five minutes old
[source,terminal]
----
$ tkn taskrun delete -all --keep-since 5
----

Reply:
Updated. Thanks.